worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -0,0 +1,35 @@
+# Dark Souls 3 Randomizer Setup Guide
+
+## Required Software
+
+- [Dark Souls 3](https://store.steampowered.com/app/374320/DARK_SOULS_III/)
+- [Dark Souls 3 AP Client](https://github.com/Marechal-L/DarkSouls3-Archipelago-client)
+
+## General Concept
+
+The Dark Souls 3 AP Client is a dinput8.dll triggered when launching Dark Souls 3. This .dll file will launch a command 
+prompt where you can read information about your run and write any command to interact with the Archipelago server.  
+
+The randomization is performed by the AP.json file, an output file generated by the Archipelago server.
+
+## Installation Procedures
+
+**This client has only been tested with the Official Steam version of the game (v1.15/1.35) not matter which DLCs are installed**
+
+Get the dinput8.dll from the [Dark Souls 3 AP Client](https://github.com/Marechal-L/DarkSouls3-Archipelago-client).  
+Then you need to add the two following files at the root folder of your game 
+( e.g. "SteamLibrary\steamapps\common\DARK SOULS III\Game" ):  
+- **dinput8.dll**
+- **AP.json** renamed from the generated file AP-{SEED_NUMBER}-{PLAYER_NUMBER}-{PLAYER_NAME}.json
+
+## Joining a MultiWorld Game
+
+1. Run DarkSoulsIII.exe or run the game through Steam
+2. Type in /connect {SERVER_IP}:{SERVER_PORT}
```

Comment:
where do you enter it? If it's a separate window (which i think it is), or if the player has to open a console, please say so.

Reply:
A separate "Windows Command Prompt" is attached to the DarkSoulsIII.exe process at startup and launched automatically.
It may not be very convenient if the player only has one screen but it works pretty well.

Reply:
Would you mind putting that into the readme? Something like
> Type /connect ... in the "Windows Command Prompt" that opened

so that there is less room for error